coriolis/osmorphing/base.py

```diff
@@ -96,6 +100,38 @@ def get_packages(self):
 
         return add, remove
 
+    def run_user_script(self, user_script):
+        if len(user_script) == 0:
+            return
+
+        script_path = "/tmp/coriolis_user_script"
+        try:
+            utils.write_ssh_file(
+                self._conn,
+                script_path,
+                user_script)
+        except Exception as err:
+            LOG.exception(err)
+            raise exception.CoriolisException(
+                "Failed to copy user script to target system."
+                " Error was: %s" % err)
```

Comment:
NIT: Considering we decided to only support Python 3, you can remove the `LOG.error` call and just say `raise exception.CoriolisException(...) from err` to preserve both stack traces together